### ISSUE TITLE
timer error connect_check_clear pc_soulball_timer

### ISSUE THREAD

**Issue by sader1992** [rathena/rathena#4907]
* **rAthena Hash**: 9794aa2b8010587af7640b7e7b7725bdf380daca

* **Client Date**: 20180919

* **Server Mode**: renewal

* **Description of Issue**: 
 I didn't see any critical problem in the server related to this issue , I should listen to the players more but they edit the skill in their mind whatever it suit them xD
will debug and add more info when I work on the server maintenance

  * Result: getting the following errors
(Those are ALL the errors(in order) I got from the time I run the server to now)
```
(05/05/2020 00:07:49) [ Error ] : delete_timer error : function mismatch (nil)(unknown timer function) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 02:19:38) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 02:19:38) [ Error ] : delete_timer error : function mismatch (nil)(unknown timer function) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:02:46) [ Error ] : delete_timer error : function mismatch (nil)(unknown timer function) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
(05/06/2020 23:06:50) [ Error ] : delete_timer error : function mismatch 0x555555f9635d(connect_check_clear) != 0x555555c87542(pc_soulball_timer)
```
  * Expected Result: no error
  * How to Reproduce: I didn't check yet (*it might be from my edits*)

* **Modifications that may affect results**: 
Sorry ,I have a lot of modification, I gathered them in the old issue post
https://github.com/rathena/rathena/issues/4856#issuecomment-620802558

**Comment by aleos89** [comment]
Have you applied this: 6570489a84c554e2fba289de325d48a35d586996

**Comment by sader1992** [comment]
yes this applied to the server

**Comment by aleos89** [comment]
Possibly fixed with f481cc9c46152b98e8166f67c1311fd12ddfc432.
